oidc/clientassertion/algorithms.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package clientassertion
+
+import (
+	"crypto/rsa"
+	"fmt"
+)
+
+type (
+	// HSAlgorithm is an HMAC signature algorithm
+	HSAlgorithm string
+	// RSAlgorithm is an RSA signature algorithm
+	RSAlgorithm string
+)
+
+const (
+	// JOSE asymmetric signing algorithm values as defined by RFC 7518.
+	// See: https://tools.ietf.org/html/rfc7518#section-3.1
```

Comment:
If you put this comment above the `const` block it will apply to all the consts, I think.